### PR TITLE
Edited content for figure 3

### PR DESCRIPTION
**Pull request by arielah**
There seems to be something wrong with 20.results.md (a merge conflict?), but I'm not sure how to fix it.

### CODE REVIEW

File: content/20.results.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ We found that white honorees have been significantly overrepresented and Asian h
 
 ### Predicting Nationality with LSTM Neural Networks and Wikipedia
```
Comment (cgreene) [inline content/20.results.md:61]:
```suggestion

We next aimed to predict the nationality of honorees and authors.
We constructed a training dataset with more than 700,000 name-nationality pairs by parsing English-language Wikipedia.
We trained a LSTM neural network on n-grams to predict nationality.
We found similar performance across 1, 2, and 3-grams; however, the classifier required fewer epochs to train with 3-grams so we used this length in the model that we term Wiki2019-LSTM.
```

File: content/20.results.md [hunk 2]
```diff
@@ -59,9 +59,9 @@ We found that white honorees have been significantly overrepresented and Asian h
...
 
-We achieved a high overall accuracy on this test data with similar precision-recall values over the classes as we saw for our Wiki2019 dataset's test set, giving confidence in our model.
-We also tested the accuracy of the LSTM trained on the ethnicolr Wiki2009 dataset in predicting the labels for unseen names in our Wiki2019 dataset.
-Importantly, this model did not achieve as high an accuracy as the model trained on Wiki2019 and tested on either dataset, suggesting our new, larger dataset was an improvement.
+Our Wiki2019-LSTM returns, for each given name, a probability of that name originating from each of our 8 regions.
+The classifier returned a multiclass ROC/AUC score of 0.954, indicating we were able to recapitulate name origin with high sensitivity and specificity.
```
Comment (cgreene) [inline content/20.results.md]:
```suggestion
We observed a multiclass ROC/AUC score of 0.954 for the classifier, indicating that the classifier can recapitulate name origin with high sensitivity and specificity.
```

File: content/20.results.md [hunk 3]
```diff
@@ -59,9 +59,9 @@ We found that white honorees have been significantly overrepresented and Asian h
...
+AUC calculated for each region individually was also high, meaning that our classifier was able to perform well on all regions (Fig. {@fig:wiki2019_lstm}).
```
Comment (cgreene) [inline content/20.results.md]:
```suggestion
AUC calculated for each region individually was also high (Fig. {@fig:wiki2019_lstm}), suggesting that our classifier was sufficient for use in a broad-scale examination of disparities.
```

File: content/40.resource-availability.md [hunk 4]
```diff
@@ -2,3 +2,4 @@
 
 A manubot instance that hosts version history for this manuscript is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity-manuscript>.
 Our analysis of authors and ISCB-associated honorees is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity>, with source code also distributed under a BSD 3-Clause License.
+Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code also distributed under a BSD 3-Clause License.
```
Comment (cgreene) [inline content/40.resource-availability.md]:
```suggestion
Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code to construct the dataset available under a BSD 3-Clause License.
```

**Review by cgreene — APPROVED** [review]
Some suggested changes but I approve when these get merged (or modified and merged) 😁

File: content/40.resource-availability.md (hunk 4 above)
Comment (dhimmel) [reply content/40.resource-availability.md]: Note that this is not currently the case. Issue still open at https://github.com/greenelab/wiki-nationality-estimate/issues/3

File: content/10.methods.md [hunk 5]
```diff
@@ -116,12 +116,27 @@ We were able to define a name and nationality for 708,493 people by using the un
...
 We initially grouped names by continent, but later decided to model our categorization after the hierarchical nationality taxonomy used by [NamePrism](http://www.name-prism.com/about) [@doi:10.1145/3132847.3133008].
 Consequently, we used the following categories: Hispanic (including Latin America and Iberia), African, Israeli, Muslim, SouthAsian, EastAsian, European (non-British, non-Iberian), and CelticEnglish (including United States, Canada, and Australia).
+Table @tbl:example_names shows the size of the training set for each of these regions, as well as examples of PubMed author names which classified to that region with greater than 95% probability.
 Our dataset, which we term Wiki2019, is available at <https://github.com/greenelab/wiki-nationality-estimate/master/data/annotated_names.tsv>.
+Full information about which countries comprised each region can be found at <https://github.com/greenelab/wiki-nationality-estimate/master/data/country_to_region.tsv>.
+
+| Region | Training Size | Sample Names |
+| ------ | ------------- | ------------ |
+| CelticEnglish | 280644 | Julie S. Miller, Jesse A. Livezey, Jeremy C Simpson, Chris Smith, Thomas M Drudge |
```
Comment (dhimmel) [inline content/10.methods.md]: In the Wiki2019-LSTM figure, we add a space between works in the regions, and we should probably do that here too.

File: content/10.methods.md [hunk 6]
```diff
@@ -116,12 +116,27 @@ We were able to define a name and nationality for 708,493 people by using the un
...
+| European | 188918 | Sven Poths, Céline Feillet, Frederik Otzen Bagger, Lars I. Leichert, Sebastian MB Nijman |
+| Hispanic | 66391 | Beatriz Peñalver Bernabé, Diego Miranda-Saavedra, Marcelo Lobosco, Euler Guimarães Horta, Edgar E Vallejo-Clemente |
+| EastAsian | 54197 | Jee-Hyub Kim, Yoriko Takahashi, Xiaohua Xu, Xuehai Zhang, Yoshihiro Noguchi |
+| Muslim | 30703 | Mohammad R. K. Mofrad, Fikret Ercal, Mehdi Yousfi Monod, Ghazaleh Taherzadeh, Noora Al Muftah |
+| SouthAsian | 20025 | Mahender Kumar Singh, Vidhu Choudhary, Suraj Pradhan, Ramakant Sharma, Vinod Menon |
+| African | 16105 | Samuel A Assefa, Nyaradzo M. Mgodi, Stanley Kimbung Mbandi, Oyebode J Oyeyemi, Ezekiel Adebiyi |
+| Israeli | 4549 | Tal Vider-Shalit, Itsik Pe'er, Michal Lavidor, Yoav Gothilf, Dvir Netanely |
+ 
+Table: Example of PubMed author names labeled to each region with high probability. {#tbl:example_names}
```
Comment (dhimmel) [inline content/10.methods.md]:
```suggestion
Table: Example of author names labeled to each region with high probability. {#tbl:example_names}
```

File: content/40.resource-availability.md [hunk 7]
```diff
@@ -2,4 +2,4 @@
 
 A manubot instance that hosts version history for this manuscript is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity-manuscript>.
 Our analysis of authors and ISCB-associated honorees is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity>, with source code also distributed under a BSD 3-Clause License.
-Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code also distributed under a BSD 3-Clause License.
+Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code to construct the dataset available under a BSD 3-Clause License.
```
Comment (dhimmel) [inline content/40.resource-availability.md]:
```suggestion
Our Wikipedia name dataset is dedicated to the public domain under CC0 License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code to construct the dataset available under a BSD 3-Clause License.
```

**Pull request merged**